universal-application-tool-0.0.1/app/models/Applicant.java

```diff
@@ -14,12 +17,15 @@
 import services.applicant.ApplicantData;
 
 /** The ebean mapped class that represents an individual applicant */
+@History
```

Comment:
question for @bion: do we want to maintain history for applicants? 

for @syntapy: I may have linked to a misleading doc in the issue, I apologize! I think here we just need the `@WhenCreated` below, but don't need to maintain history on the Applicant table. I'm not sure, though, so I'm looping in Bion

Reply:
Having timestamps for Applicants is good but we do not need history for applicants. Adding history increases scope for the work to be compliant for data retention policies so it's best to leave it out.